client/sites-dashboard/components/sites-p2-badge.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from '@emotion/styled';
+
+const SitesLaunchStatusBadge = styled.span`
+	font-size: 12px;
+	font-weight: 400;
+	color: #fff;
+	background-color: #2b65f6;
+	padding: 0px 10px;
+	border-radius: 4px;
+	line-height: 20px;
+	display: inline-block;
+`;
```

Comment:
This component is the same as `SitesLaunchStatusBadge`, so why not extend from it?

```suggestion
const SitesP2Badge = styled(SitesLaunchStatusBadge)`
	color: #fff;
	background-color: #2b65f6;
`;
```

Reply:
> This component is the same as `SitesLaunchStatusBadge`, so why not extend from it?

Sure, did with 4e6aae1e18